Jetchat/app/src/main/java/com/example/compose/jetchat/components/JetchatScaffold.kt

```diff
@@ -16,25 +16,25 @@
 
 package com.example.compose.jetchat.components
 
-import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.material3.DrawerState
+import androidx.compose.material3.DrawerValue.Closed
 import androidx.compose.material3.ExperimentalMaterial3Api
-import androidx.compose.material3.Scaffold
-import androidx.compose.material3.ScaffoldState
-import androidx.compose.material3.rememberScaffoldState
+import androidx.compose.material3.NavigationDrawer
+import androidx.compose.material3.rememberDrawerState
 import androidx.compose.runtime.Composable
 import com.example.compose.jetchat.theme.JetchatTheme
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun JetchatScaffold(
```

Comment:
We should probably rename this as it's not a `Scaffold` any more?

Reply:
https://github.com/android/compose-samples/issues/720